ballerina/schemaClient.bal

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We'll have to use the new license header
https://github.com/ballerina-platform/module-ballerina-persist/blob/main/compiler-plugin-test/src/test/resources/test-src/plugin/package_24/sample.bal